pkg/kubelet/api/v1alpha1/runtime/api.pb.go

```diff
@@ -0,0 +1,2778 @@
+/*
```

Comment:
Spoke to @vishh offline. I filed #28642 to discuss handling the API machinery in general for the long term. For now, I think it's ok to check in generated go code.